content/using.html

```diff
@@ -414,16 +418,16 @@
 
 Now, upload your test package to Anaconda Cloud using the [anaconda-client upload](cli.html#Upload) command.
 
-The `--channel` option tells Anaconda Cloud not to put the upload into your `main`
-channel where all users will be able to install it.
+Adding the `--label` option tells Anaconda Cloud to make the upload visible 
+only to users who specify that label.
 
-    anaconda upload #userdefined{/path/to/conda-package-2.0.tar.bz2} --channel #userdefined{test}
+    anaconda upload #userdefined{/path/to/conda-package-2.0.tar.bz2} --label #userdefined{test}
 
-You will notice now that even when you search your conda `main` channel,
+You will notice now that even when you search conda `main`,
 you won't see the `2.0` version of the test package. This is because you have to tell conda
-to look in your new `test` channel.
+to look for your new `test` label.
```

Comment:
This is where we should be very careful. conda is still using the channels terminology.

perhaps:


You will notice now that even when you search conda this looks only for files labeled `main`.
Because of this, you won't see the `2.0` version of the test package. You have to tell conda		 
to look in your channel and select files that are labeled `test`.